src/components/include/cobj_format.h

```diff
@@ -38,23 +38,21 @@
 #define COBJ_NAME_SZ 64
 
 /* cobj flags */
-enum {
-	COBJ_INIT_THD = 1,
+enum { COBJ_INIT_THD = 1,
```

Comment:
UR.  We'd like to just keep enums on separate lines, but if it can't do that, then fine.  However, wow, the `}` on a separate line?  UR.